src/arrayMethodSort.js

```diff
@@ -4,8 +4,26 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
Using `__proto__` to extend the array prototype is not recommended. Instead, consider using `Array.prototype` to define the `sort2` method. This is a more standard and safer approach.